addon/styles/addon.scss

```diff
@@ -59,8 +63,8 @@ $z-index-modal-hidden: -1 !default;
   .nav-modal {
     display: flex;
     position: fixed;
-    top: 50px;
-    left: 0;
+    top: $frost-navigation-bar-height;
+    left: 0 !important; // sass-lint:disable-line no-important
```

Comment:
Can you just add a comment here explaining why important is being used (liquid-fire)